core/vm/gas.go

```diff
@@ -30,6 +30,18 @@ const (
 	GasExtStep     uint64 = 20
 )
 
+// createGas returns the actual gas available for the call.
+//
+// The cost of gas was changed during the homestead price change HF.
+// As part of EIP 150 (TangerineWhistle), the returned gas is gas - gas * 63 / 64.
+func createGasEip150(isEip150 bool, availableGas uint64) (uint64, error) {
```

Comment:
Please review the whole logic behind [commit](https://github.com/ethereum/go-ethereum/pull/21832/commits/f0804e25d6a10a458584cf06067080061d44389a), as anything wrong across the lines might bring consensus issues.